sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -243,7 +243,7 @@ void gnrc_rpl_parent_update(gnrc_rpl_dodag_t *dodag, gnrc_rpl_parent_t *parent)
             kernel_pid_t if_id;
             if ((if_id = gnrc_ipv6_netif_find_by_addr(NULL, &all_RPL_nodes)) != KERNEL_PID_UNDEF) {
                 fib_add_entry(&gnrc_ipv6_fib_table, if_id, def.u8, sizeof(ipv6_addr_t),
-                              (FIB_FLAG_NET_PREFIX | 0x0),
+                              (0x00 << FIB_FLAG_NET_PREFIX_SHIFT | 0x0),
```

Comment:
Why don't you just write `0x0` instead of `0x00 << FIB_FLAG_NET_PREFIX_SHIFT | 0x0` ?